docs/studios/unity/tutorials/quickstart_sample_scene.mdx

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_label: 'Quickstart - Sample Scene and Bot'
+---
+
+# Quickstart - Sample Scene and Bot
+
+**_Time to complete: 2 minutes_**
```

Comment:
I would remove this altogether or make it say 15 minutes.  You can't realistically read this page and understand it in 2 minutes, even without following the steps it lists.  

Downloading unity projects or starting them for the first time, downloading our sdk, etc all will definitely take longer than that.

Reply:
Good point re 2 minutes being too short, will change this to ten minutes I think - I just did a run through myself to test this, and even with a bunch of Unity-caused errors and reading through the text of the guide, it took about 6 minutes and 30 seconds. Will increase it to 10 minutes due to the time it may take to download URP. I think it is realistic that I can read the page and understand in less than 10 minutes at least (note they shouldn't leave this with any understanding of how the bot works or is integrated, that is only for the next tutorial, this is just to get a bot up and running).

Here is the loom video: https://www.loom.com/share/de090ddd5a1544b8a62ea52869aadd8a